src/Command/Integration/IntegrationCommandBase.php

```diff
@@ -344,6 +344,7 @@ private function getFields()
                 ]],
                 'description' => 'The From address for alert emails',
```

Comment:
```suggestion
                'description' => '[Optional] Custom From address for alert emails',
```
What do you think about something along these lines @pjcdawkins ? I'd like the description text to help folks understand they can add a custom one, but that it's not required :)

Reply:
Yes good point